index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
True. However, if we're queing into the global performance buffer, it only seems natural that these entries are queryiable? In fact, I think that directly falls out of the processing algorithm.

/cc @toddreifsteck